docker-compose.dev-thirdparty.yml

```
@@ -0,0 +1,429 @@
+version: "3.7"
+#https://github.com/compose-spec/compose-spec/blob/master/spec.md#using-extensions-as-fragments
+x-logging: &default-logging
+  options:
+    max-size: "100m"
+    max-file: "5"
+  driver: json-file
+x-version:
+  &chaosgenius-version
+  CHAOSGENIUS_VERSION_POSTFIX=stable-thirdparty-compose
```

This should be named `dev-thirdparty-compose`
```suggestion
  CHAOSGENIUS_VERSION_POSTFIX=dev-thirdparty-compose
```